server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk

```diff
@@ -126,8 +126,8 @@
               <p class="govuk-body">Reload the page or try again later. Other parts of this service may still be available.</p>
             {% endif %}
               <p class="govuk-body govuk-!-margin-top-2 govuk-!-display-none-print">
-                <a class="govuk-link" href="../education-and-training" data-qa="view-all-courses-button">
-                  View all education and training<span class="govuk-visually-hidden">functional skills</span>
+                <a class="govuk-link" href="/plan/{{ prisonerSummary.prisonNumber }}/view/education-and-training" data-qa="view-education-and-training-tab-button">
```

Comment:
This link now navigates to the Education and Training tab (as required in the ticket).